tests/databrowser.js

```diff
@@ -401,7 +403,7 @@ define([
             it("overwrites the forms preloaded tags", function() {
                 var form = call('getData').core.form;
                 assert(form.isValidHashtag(dobProp));
-                assert.strictEqual(form.hashtagMap[dobProp], null);
+                form.hashtagMap[dobProp] = null;
```

Comment:
No, since xpaths are now inferred on loading the form the path is not null at this point. I had to set it to null here so I could verify that `loadCaseData` actually updated `hasthagMap` (because it loads the same path as the one that is inferred on loading the form).